### PR TITLE
Bug 212720 - JFace viewer for PShelf

### PR DESCRIPTION
**Pull request by lcaron**
Apply viewer developed by  Richard Michalsky

### CODE REVIEW

File: widgets/opal/commons/org.eclipse.nebula.widgets.opal.commons/src/org/eclipse/nebula/widgets/opal/commons/ReflectionUtils.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package org.eclipse.nebula.widgets.opal.commons;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class ReflectionUtils {
```
Comment (wimjongman) [inline widgets/opal/commons/org.eclipse.nebula.widgets.opal.commons/src/org/eclipse/nebula/widgets/opal/commons/ReflectionUtils.java:6]: Missing javadoc for API

Comment (lcaron) [reply widgets/opal/commons/org.eclipse.nebula.widgets.opal.commons/src/org/eclipse/nebula/widgets/opal/commons/ReflectionUtils.java:6]: Ok. Should I fix it before the XMas release or could I work on this next week ?

**Comment by wimjongman** [comment]
Fine for later. Thanks!

**Comment by lcaron** [comment]
I had time in this cold and rainy day, it is fixed

**Pull request merged**